test/integration/driver.go

```diff
@@ -201,7 +201,30 @@ func deploy(name, svcName, dType, port1, port2, port3, port4, version string) er
 		return err
 	}
 
-	if err := w.Flush(); err != nil {
+	writer := bufio.NewWriter(f)
+	if injectProxy {
+		p := &inject.Params{
+			InitImage:        fmt.Sprintf("%s/init:%s", params.hub, params.tag),
+			RuntimeImage:     fmt.Sprintf("%s/runtime:%s", params.hub, params.tag),
+			RuntimeVerbosity: 2,
```

Comment:
can you add a verbosity field at the top since it's used in two places, here and in `write` function?